server/game/cards/locations/04/seabitch.js

```diff
@@ -0,0 +1,41 @@
+const DrawCard = require('../../../drawcard.js');
+ 
+class SeaBitch extends DrawCard {
+    setupCardAbilities() {
+        this.action({
+            title: 'Sacrifice this card',
+            method: 'sacrifice'
+        });
+    }    
+
+    sacrifice(player) {
+        if(this.location !== 'play area') {
+            return false;
+        }
+        this.game.promptForSelect(player, {
+            cardCondition: card => this.cardCondition(card),
+            activePromptTitle: 'Select location to take control of',
+            waitingPromptTitle: 'Waiting for opponent to use ' + this.name,
+            onSelect: (player, card) => this.onCardSelected(player, card)
+        });
+    }
+
+    cardCondition(card) {
+        return card.getType() === 'location' && card.location === 'play area' && card.owner !== this.owner && !card.hasKeyword('Limited');
+    }
+
+    onCardSelected(player, card) {      
+        this.untilEndOfPhase(ability => ({
+            targetController: 'opponent',
+            match: target => target === card,
```

Comment:
Since we have a specific card object here (the `card` parameter from the select prompt), then `match: card,` should probably be preferred. Basically - if you have a single card you're applying the effect to, `match` should be the card object. Otherwise, it should be a function that returns `true` when a card should receive the effect.